src/components/Layout/Header/NavBar/Notifications.tsx

```diff
@@ -150,7 +152,7 @@ export const Notifications = memo(() => {
           disableAnalytics={disableAnalytics}
         >
           <NotificationFeed gapFromBell={10} placement='bottom-end'>
-            <IconButton aria-label='Open notifications'>
+            <IconButton aria-label={translate('navBar.openNotif')}>
```

Comment:
Very nitpicky but prefer camelCasing the translation keys when possible for grepping purposes i.e `Open Notifications` -> `openNotifications` but no need to tackle here

Reply:
Fair enough, I was aiming for shorter but still understandable with camelCase :D